client/landing/stepper/declarative-flow/internals/steps-repository/domains/style.scss

```diff
@@ -444,6 +444,72 @@
 
 }
 
+.hundred-year-domain.step-route {
+	margin-top: -60px;
+}
+
+.hundred-year-domain.domains {
+	.step-container__header {
+		.formatted-header {
+			.formatted-header__title {
+				font-size: rem(32px);
+				@include break-small {
+					font-size: rem(44px);
+				}
+			}
+			@include break-small {
+				margin-top: 60px;
+				margin-bottom: 0;
+			}
+			margin: 32px 16px;
+		}
+
+		@include break-small {
+			margin-bottom: 40px;
+		}
+	}
+
+	.domains__content {
+		width: 100%;
+		@include break-small {
+			max-width: 960px;
+		}
+
+		.domains__step-content {
+			padding: 0 16px;
+			@include break-small {
+				padding: 0 48px 0 32px;
+			}
+		}
+		// Hide the "Sale" tag
+		.badge.badge--warning {
```

Comment:
This raises a flag for me. I don't have any context on how the Sale tag works at the moment, but my first thought would be to remove the tag instead of hiding it. I'm wondering if the Sale tag is related to the product_id, and if this is a temp approach?

Reply:
I agree with you 🙂 the sale tag is shown for domains with TLDs that have an active sale (see the `sale-coupons` MC tool) and are sold for a lower price than normal. I believe these domains should not be shown in the 100-year flows, so this CSS would actually hide that issue.

This was actually copy-pasta from the 100-year plan CSS. I think we don't even need this style since com/net/org domains don't have sales for now, so I'll remove this.